### ISSUE TITLE
windows command line is very slow when lively is running

### ISSUE THREAD

**Issue by cngxx** [rocksdanister/lively#1422]
**Describe the bug**
Hello i am using batch *.bat files through cmd.exe for my work and when lively is runing(even if wallpaper is stopped due to unfocused desktop) the cmd.exe is working very slow(its like each line gets processed every x minutes). Other software is working fine, these batch files has no impact on performance(its simple tasks) also cpu and gpu isnt utilizes much(up to 20%). When i close entire app i become fine.

**To Reproduce**
1. activate any live wallpaper
2. make simple *.bat batch file with few lines(like hello world) and run it

**Expected behavior**
Command line working at same speed when lively is or isnt running.

**Screenshots/Video**
I can make some video but it will be the same but one will be longer.

**Desktop (please complete the following information):**
 - OS: Windows 10 21H2 19044.1889
- All wallpapers doing it, no matter if its running or it is stopped due to unfocused desktop.

**Additional context**
I think i described it enough so i dont know what to write any more.

**Log file (Important)**
Share the diagnostic files by:
Open Lively settings (Gear icon) top-right -> System -> Log File -> Save
or 
Navigate to Appdata folder `C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs`
[lively_log_20221013_102705.zip](https://github.com/rocksdanister/lively/files/9772437/lively_log_20221013_102705.zip)

**Comment by rocksdanister** [comment]
Looking at the logs, only thing that I see as unique is:
`Citrix Indirect Display Adapter`

Quick search shows forum posts with graphic issues:
https://steamcommunity.com/discussions/forum/1/1743358239830363122/?l=latam

If possible try disabling and see?

Can you give some example bat file? I have no issue with cmd.

**Comment by cngxx** [comment]
Thanks for your time, im sorry but it has to be something in my notebook, it was few days acting as i described(with lively running or not) but now it is doing strange random slows only for few remote bat files, which are still affected by running lively. I tried another PC and its doing fine. I will make some research when ill have more time. Thanks, have a nice day.
P.S.: I tried uninstalling Citrix workspace but no change in behavior.
